app/controllers/events/view/edit/attendee.js

```diff
@@ -0,0 +1,27 @@
+import Controller from '@ember/controller';
+import EventWizardMixin from 'open-event-frontend/mixins/event-wizard';
+
+export default Controller.extend(EventWizardMixin, {
+  async saveForms(data) {
+    for (const customForm of data.customForms ? data.customForms.toArray() : []) {
+      await customForm.save();
+    }
+    return data;
+  },
+  actions: {
+    save(data) {
+      this.saveForms(data);
```

Comment:
Why are you not handling the errors ?

Reply:
@niranjan94 The errors are being handled by the functions `saveForms` and `saveEventDataAndRedirectTo` themselves.

Reply:
No. They are not. I'm not talking about the form validation errors. Lets say `customForm.save` fails with an error. How is that being handled ?

Reply:
@uds5501 maybe you can use try-catch to handle such errors

Reply:
@niranjan94 @shreyanshdwivedi Alright, will work on it tonight